client/my-sites/plugins/controller.js

```diff
@@ -144,6 +145,35 @@ function renderProvisionPlugins() {
 }
 
 controller = {
+	validateFilters: function( filter, context, next ) {
+		const wpcomFilters = [ 'standard', 'business', 'premium' ];
```

Comment:
It doesn't change much, but makes sense to remove those and add them when we need them.
BTW, Done!